AISKU/src/Initialization.ts

```diff
@@ -110,6 +111,7 @@ export class Initialization implements IApplicationInsights {
 
     private dependencies: DependenciesPlugin;
     private properties: PropertiesPlugin;
+    private sender: Sender;
```

Comment:
Can we rename this to ```_sender```, I just like to call out private properties with the prefix '_' so that when reading later code it's obvious.

renaming the dependencies and properties, would be nice --- but you may have to update tests if you do that.
Eventually, this will probably be converted to use dynamicProto() and the privates become internal properties that will not be exposed.